core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotReferenceCreationValidator.scala

```diff
@@ -30,12 +30,17 @@ class SnapshotReferenceCreationValidator(val workspaceContext: Workspace, val sn
   // Throws an exception when the given snapshot cannot be referenced by the given workspace due to crossing an
   // unsupported platform boundary.
   @throws(classOf[PlatformBoundaryException])
-  def validateWorkspacePlatformCompatibility(workspacePlatform: WorkspaceCloudPlatform): Unit =
+  def validateWorkspacePlatformCompatibility(workspacePlatform: Option[WorkspaceCloudPlatform]): Unit =
```

Comment:
This is to handle the single usage by SnapshotService, at line

Reply:
FYI @jladieu ^

Reply:
I'm having some trouble writing a test case for line 39. The test case that covers line 47 requires that a workspace actually exist (and it's in Ready state)-- it is using `testData.azureWorkspace`.

Reply:
Ah never mind, I figured it out. I just pushed up a test case.